ThermofluidStream/Boundaries/Reservoir.mo

```diff
@@ -1,15 +1,15 @@
 within ThermofluidStream.Boundaries;
-model Reservoir "Model of a Resorvoir"
+model Reservoir "Model of a Reservoir"
   extends Internal.PartialVolume(final useHeatport = false, final initialize_pressure=false, final p_start=0, final A = 0, final U=0);
 
-  parameter Boolean pEnvFromInput = false "Enable Input";
+  parameter Boolean pEnvFromInput = false "Enable input";
   parameter SI.Area A_surf(displayUnit="cm2")=0.01 "Base area of medium";
-  parameter SI.Pressure p_env_par=1e5 "Environment Pressure"
+  parameter SI.Pressure p_env_par=1e5 "Environment pressure"
```

Comment:
```suggestion
  parameter SI.Pressure p_env_par=1e5 "Environmental pressure"
```
In general I think "ambient pressure" would be the better term to use but then we'd have to change the name of the parameter (and conversion script) as well...

Reply:
Changing seems like a good idea for consistency. Currently both ambient and environment(al) are used in different places.

Reply:
Mhh yes then we should change it to be consistent.